lib/story_branch.rb

```diff
@@ -203,8 +203,10 @@ def story_finish
 
     def config_value key, env
       PIVOTAL_CONFIG_FILES.each do |config_file|
-        pivotal_info = YAML.load_file config_file
-        return pivotal_info[key] if pivotal_info[key]
+        if File.exists? config_file
```

Comment:
Else does not need to do anything. It is looking through the different possible config files for the config key. If the file does not exist moves on to the next one.
If the config is not found in any config file, it falls back to the environment variables, which throw the expected error on env_required method